src/savegame_json.cpp

```diff
@@ -859,6 +859,18 @@ void npc::load(JsonObject &data)
         attitude = npc_attitude(atttmp);
     }
 
+    if ( data.read( "companion_mission", comp_miss) ) {
```

Comment:
I'm continuing the convention for the other variables that appear to have been added.  I don't have the time/resources to test what will be compatible with saves from previous versions.  Just went with the safe bet.  I'll fix it.